experiment/process-e2e-suite-log/endpoints.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+var endpoints = map[string]string{
```

Comment:
How is this map generated? How do we keep it up to date when the API changes?

Reply:
The map is the result of these steps.
- selecting endpoints to measure, picking the http verb and the uri.
- creating the regular expressions manually. This involved looking at how they appeared in the master log (For example "GET" over a resource exposing a collection appears translated as "list" in the master log).
- placing both the original data (http verb + uri) as a key, and as the correspondent regex as value.

This effort was originally meant to look only at a sub-selection of endpoints (not the whole kubernetes api spec) so we could analyse by section and learn from the report. So the "endpoints.go" file should change encompassing the endpoints selected for analysis (its an input for the process). Let me know if there are easier/better options to deal with this. Thanks a lot!